grpc-native/src/main/java/org/ballerinalang/net/grpc/listener/UnaryServerCallHandler.java

```diff
@@ -37,9 +37,15 @@
 
     public UnaryServerCallHandler(Descriptors.MethodDescriptor methodDescriptor, ServiceResource resource)
             throws GrpcServerException {
+
         super(methodDescriptor);
         if (resource == null) {
-            throw new GrpcServerException("Unary service resource doesn't exist.");
+            String serviceType = "Unary";
```

Comment:
```suggestion
            String serviceType = "Simple";
```

Reply:
Shall we change this to `simple`. word `unary` service is now replaced with `simple`

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-grpc/pull/314/commits/b77f14c2e5ad6283dbbf3be11413a1cce1007824